libretro/libretro.go

```diff
@@ -452,6 +454,11 @@ func coreInputPoll() {
 	inputPoll()
 }
 
+//export SetControllerPortDevice
+func (core *Core) SetControllerPortDevice(port uint, device uint) {
+	C.bridge_retro_set_controller_port_device(core.symRetroSetControllerPortDevice, C.unsigned(port), C.unsigned(device))
+}
```

Comment:
You're mixing two things here. There are exported functions like `coreInputState` that start by a lowecase character and are not applied on the Core type. And there are functions like `SetInputState` that start with an uppercase character and are applied to the Core type.

cgo is everything except user friendly ;)